packages/in_app_purchase/in_app_purchase_ios/test/fakes/fake_ios_platform.dart

```diff
@@ -165,6 +181,11 @@ class FakeIOSPlatform {
               createFailedTransaction(id);
           InAppPurchaseIosPlatform.observer
               .updatedTransactions(transactions: [transaction_failed]);
+        } else if (testTransactionCancel > 0) {
+          SKPaymentTransactionWrapper transaction_canceled =
+              createCanceledTransaction(id, testTransactionCancel);
+          InAppPurchaseIosPlatform.observer
+              .updatedTransactions(transactions: [transaction_canceled]);
```

Comment:
Also typos in the word "cancelled":
```suggestion
          SKPaymentTransactionWrapper transaction_cancelled =
              createCancelledTransaction(id, testTransactionCancel);
          InAppPurchaseIosPlatform.observer
              .updatedTransactions(transactions: [transaction_cancelled]);
```